Rock/Model/Person.cs

```diff
@@ -1010,6 +1010,30 @@ private set
         }
 
         /// <summary>
+        /// Returns a formatted age in years, months, or days.
+        /// </summary>
+        /// <returns></returns>
+        public string FormatAge()
+        {
+            if (Age > 0)
+            {
+                return Age + (Age == 1 ? " yr old " : " yrs old ");
+            }
```

Comment:
Also, the `Age` property on person is not a stored value. It does an age calculation from the birthdate property whenever it is referenced. Because you're referencing this property three times, it would be better to only use it once and save it to a local int? variable before doing the comparisons. Also, it may return a null value, be sure to check for that.